src/stories/Library/nav-teaser/nav-teaser.scss

```
@@ -1,5 +1,6 @@
 .nav-teaser {
   @include typography($typo__body-placeholder);
+  color: #000;
```

```suggestion
  color: $color__global-black;
```

Please use the available color variables.